examples/serial_com.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""
+This example sends every second a messages over the serial interface and also 
+receives incoming messages.
+
+python3 -m examples.serial_com
+
+Expects two serial ports (/dev/ttyS10 and /dev/ttyS11) connected to each other:
+    Linux:
+    To connect two ports use socat.
+    sudo apt-get install socat
+    sudo socat PTY,link=/dev/ttyS10 PTY,link=/dev/ttyS11
+
+    Windows:
+    This example was not tested on Windows. To create and connect virtual
+    ports on Windows, the following software can be used:
+        com0com: http://com0com.sourceforge.net/
+"""
+
+import time
+import can
+import threading
+
+
+def time_millis():
+    return int(round(time.time() * 1000))
+
+
+def send_cyclic(bus, msg, stop_event):
+    print("Start to send a message every 1s")
+    start_time = time_millis()
+    while not stop_event.is_set():
+        msg.timestamp = time_millis() - start_time
```

Comment:
The timestamp is defined in seconds so this is incompatible with the rest of interfaces.